xla/service/gpu/tests/BUILD

```diff
@@ -807,15 +807,12 @@ lit_test_suite(
     args = if_cuda_is_configured([
         "--param=PTX=PTX",
         "--param=GPU=a6000",
-        "--param=GPU1=v100",
-        "--param=SM=CHECK-SM70",
-        "--param=GPU2=a100_80",
+        "--param=sm70",
```

Comment:
features need to be given without "--param=" prefix. Right now, the %if condition is never triggered, as the sm70/sm80 is not recognized as a feature.
So just adding "sm70", "sm80" should work.

Reply:
Ok, seems it is actually not that easy, it also needs other changes. Currently this would result in lit: error: unrecognized arguments: sm70
I think we need to let lit know that the feature sm70 exists. I am currently trying it out myself what needs to be done.

Reply:
Ok, seems it actually needs to be passed via --param=sm70 like you did. But what is missing is lines like this:

config.available_features.add("sm70")

These need to be added to xla/lit.cfg.py

Reply:
So just summarizing, everything in your changes is already ok, just the changes to xla/lit.cfg.py need to be added (for each feature, add a line `config.available_features.add(...)`

Reply:
> config.available_features.add("sm70")
>These need to be added to xla/lit.cfg.py

Hm, just adding these features will make if statement true, regardless of what is in the BUILD file. We'd need something like this in  lit.cfg.py:

```
if using-cuda:
   config.available_features.add("sm70")
   config.available_features.add("sm80")
if using-rocm:
  config.available_features.add("if_rocm")
```

And I'm not sure how I could get target device here

Reply:
Ok, then I guess unfortunately this feature stuff doesn't work. Sad :-(
Then I guess we need to go back to your original idea.

Reply:
Reverted everything

Reply:
Thanks, and sorry for the extra work. I hoped it was possible to fix this in a better way.